src/openrct2-ui/windows/Ride.cpp

```diff
@@ -5918,7 +5918,7 @@ static void window_ride_graphs_invalidate(rct_window* w)
     window_ride_graphs_widgets[WIDX_GRAPH_ALTITUDE].top = y;
     window_ride_graphs_widgets[WIDX_GRAPH_VERTICAL].top = y;
     window_ride_graphs_widgets[WIDX_GRAPH_LATERAL].top = y;
-    y += 11;
+    y += 13;
```

Comment:
Is there a constant for this?

Reply:
Right, I'll go with `BUTTON_FACE_HEIGHT + 1` then.